### ISSUE TITLE
Cannot build Android with 9.0.0

### ISSUE THREAD

**Issue by asaarnak** [flutter-stripe/flutter_stripe#1154]
Similar issue previously:
In 7.0.0: https://github.com/flutter-stripe/flutter_stripe/issues/1022
In 8.0.0: https://github.com/flutter-stripe/flutter_stripe/issues/1103
```
e: file:///Users/builder/programs/flutter/.pub-cache/hosted/pub.dev/stripe_android-9.0.0/android/src/main/kotlin/com/reactnativestripesdk/StripeSdkModule.kt:808:14 Expression 'size' of type 'Int' cannot be invoked as a function. The function 'invoke()' is not found
e: file:///Users/builder/programs/flutter/.pub-cache/hosted/pub.dev/stripe_android-9.0.0/android/src/main/kotlin/com/reactnativestripesdk/StripeSdkModule.kt:809:127 Expression 'size' of type 'Int' cannot be invoked as a function. The function 'invoke()' is not found
```

**Comment by asaarnak** [comment]
Created PR: https://github.com/flutter-stripe/flutter_stripe/pull/1155
Temporary workaround:
```
dependency_overrides:
  stripe_android:
    git:
      url: https://github.com/asaarnak/flutter_stripe
      ref: patch-3
      path: packages/stripe_android
```

**Comment by jonasbark** [comment]
Fixed in 9.0.0+1
